cmd/fyne/internal/commands/build.go

```diff
@@ -7,16 +7,89 @@ import (
 	"strings"
 
 	version "github.com/mcuadros/go-version"
+	"github.com/urfave/cli/v2"
 )
 
-type builder struct {
+// Builder generate the executables.
+type Builder struct {
 	os, srcdir, target string
+	goPackage          string
 	release            bool
 	tags               []string
+	tagsToParse        string
 
 	runner runner
 }
 
+// Build returns the cli command for building fyne applications
+func Build() *cli.Command {
+	b := &Builder{}
+
+	return &cli.Command{
+		Name:        "build",
+		Usage:       "Build an application.",
+		Description: "You may specify the --executable to build, but --target will define the OS it is build for indiscriminate of the extention of the executable.",
```

Comment:
This documentation should likely be tidied up, in a similar fashion to https://github.com/fyne-io/fyne/pull/2821/files#r825293352.